src/Cli/dotnet/ParseResultExtensions.cs

```diff
@@ -212,7 +212,7 @@ private static IEnumerable<string> GetRunPropertyOptions(ParseResult parseResult
             var optionString = shorthand ? "-p" : "--property";
             var options = parseResult.CommandResult.Children.Where(c => c.Token().Type.Equals(TokenType.Option));
             var propertyOptions = options.Where(o => o.Token().Value.Equals(optionString));
-            var propertyValues = propertyOptions.SelectMany(o => o.Children.SelectMany(c => c.Tokens.Select(t=> t.Value))).ToArray();
+            var propertyValues = propertyOptions.SelectMany(o => o.Tokens.Select(t=> t.Value)).ToArray();
```

Comment:
`OptionResult.Tokens` returns Tokens of all its children:

https://github.com/dotnet/command-line-api/blob/01aee747e4524464bfe63b73f59bc68f9abbb33e/src/System.CommandLine/Parsing/ParseResultVisitor.cs#L195-L196